spacebench/algorithms/dapsm.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd
+import numpy as np
+import sys
+from scipy.optimize import linear_sum_assignment
+
+def StandDist(x):
```

Comment:
All function names should follow the same naming convention. We reserve CamelCase naming convention for class and snake_case for functions and variable names.

```
StandDist --> stand_dist
```